src/components/Calculator.css

```diff
@@ -1,9 +1,11 @@
 .container {
-  margin-top: 4rem;
-  margin-left: 50%;
   display: flex;
-  flex-direction: column;
-  max-width: 33.33%;
+  justify-content: space-between;
+  padding: 0 2rem;
+}
+
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md)for this project, " `The general layout should match these wireframes:` ", please make sure your calculator page matches the wireframe. Please make sure all the text and layout of your calculator page is the same as in the [wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/math_magicians_calculator.png).

Reply:
Done. Styling meets project wireframe template. Thanks.